### ISSUE TITLE
Explicit print statement in code

### ISSUE THREAD

**Issue by SqlAndWood** [nabsabraham/pandas-erd#5]
Great capability, love the code. 

Would it be possible to provide a bool input to prevent print statements from being displayed?

for example: within module.py:

   def write_to_file(self, filename='output.txt', display_output: bool = False):
        self.filename = filename
        text_file = open(self.filename, "w")
        text_file.write(self.res)
        text_file.close()
        if display_output:
           print(f'written to {self.filename}')

Thanks for the consideration.

**Comment by nabsabraham** [comment]
yes that sounds okay to me! would be happy to merge if you submit a pr :)
